src/binding/c/mpit_api.txt

```diff
@@ -357,10 +358,23 @@ MPI_T_pvar_read:
 
 MPI_T_pvar_readreset:
     .desc: Read the value of a performance variable and then reset it
+    .error: MPI_T_ERR_INVALID_HANDLE, MPI_T_ERR_PVAR_NO_ATOMIC
```

Comment:
should be part of previous commit?

Reply:
I separated both changes and then squashed back to https://github.com/pmodels/mpich/pull/4952/commits/d8fcdb7a3c391b97415151d278f2a874aaea16b4 along with previous commits (add missing error codes) since they are what missed from the original commit.